tests/system/small/test_dataframe.py

```diff
@@ -1740,6 +1755,8 @@ def test_join_duplicate_columns_raises_not_implemented(scalars_dfs):
 
 @all_joins
 def test_join_param_on(scalars_dfs, how):
+    if how == "cross":
+        pytest.skip("cross join not supported with 'on' param")
```

Comment:
probably more meaningful to assert with `pytest.raises` in this case

Reply:
done